src/Kafka/Consumer/Types.hs

```diff
@@ -67,6 +79,7 @@ data RebalanceEvent =
   | RebalanceRevoke [(TopicName, PartitionId)]
   deriving (Eq, Show, Generic)
 
+-- | The partition offset
 data PartitionOffset =
```

Comment:
The difference between a `PartitionOffset` and an `Offset` didn't strike me (of course `PartitionOffset` has other constructors, but I failed to find a good explanation of why both types exist)